models/semantic_networks.py

```diff
@@ -43,11 +42,13 @@ def define_f(
     f_s_net,
     model_input_nc,
     f_s_semantic_nclasses,
+    model_type_sam,
     model_init_type,
     model_init_gain,
     f_s_config_segformer,
     f_s_weight_segformer,
     f_s_weight_sam,
+    f_s_weight_mobile_sam,
```

Comment:
Let's use a single option, f_s_weight_sam, for both regular and mobile sam, since I don't see a use case where we would like to use both, thus one option is enough.